src/components/forms/checkbox.jsx

```diff
@@ -15,9 +15,7 @@ var Checkbox = React.createClass({
             this.props.className
         );
         return (
-            <div className={classes}>
-                <FRCCheckbox {... this.props} />
-            </div>
+            <FRCCheckbox elementWrapperClassName={classes} {... this.props} />
```

Comment:
Why did you decide to move the class to an internal part of the component?  I would expect the `className` I give to a component to be applied to the outermost element of that component so that I can target anything within the component under that `className`, e.g.
```jsx
<Checkbox className="my-unique-checkbox" />
```
```scss
.my-unique-checkbox {
    // Styles for the outermost element
    .help-block { // Or some other sibling to the element wrapper
        // ...
    }
}
```

Most importantly doing it this way makes it so you don't have to know how the checkbox component works in order to add outermost block-level styles to it (`float`, etc.). Currently, wouldn't you need to add styles to the row classname, and then it wouldn't be namespaced to the className you gave to your component?